index.html

```diff
@@ -46,7 +46,7 @@
   <!-- G+ sharing meta data -->
   <meta itemprop="name" content="{$ title $}">
   <meta itemprop="description" content="{$ description $}">
-  <meta itemprop="image" content="{$ url $}images/social-share.jpg">
+  <meta itemprop="image" content="{$ image if image.startsWith('http') else (url + image) $}">
```

Comment:
It's not good to check url like this.  How about using regex to validate if it is valid URL?

Reply:
I am not checking if it's valid url or not. What I do is to check if it's an absolute path or relative